### PR TITLE
WSTEAM1-134 - Enable YouTube consent banner on CPS STY pages

### PR DESCRIPTION
**Pull request by amoore108**
Resolves WSTEAM1-134 

**Overall change:**
Enables the consent banner for YouTube embeds on CPS story pages. Updates the consent banner to have a `GREY_5` coloured border to allow it to display properly in both CPS and Optimo pages

**Code changes:**
- Removes checks for `ARTICLE_PAGE` type to allow the consent banner to appear in both CPS and Article pages
- Adds new `GREY_5` colour to the Theme colour palette
- Updates tests to reflect allowing rendering in multiple page types
- Updated the `ConsentBanner` component to use font from Theme

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by mfonofm — APPROVED** [review]
Nice!

File: src/app/components/EmbedConsentBanner/ConsentBanner.tsx [hunk 1]
```diff
@@ -181,36 +170,15 @@ const ConsentBanner = ({
...
       </p>
       <button
         data-testid="banner-button"
-        css={[
-          // TODO: Remove custom font functions and use theme
-          consentBannerCss.button,
-          getSansBold(service),
-          getPica(script),
-        ]}
+        css={consentBannerCss.button}
```
Comment (andrewscfc) [inline src/app/components/EmbedConsentBanner/ConsentBanner.tsx:191]: loving the extraction of the styles to a new file, feels much cleaner!

File: src/app/legacy/containers/SocialEmbed/index.test.jsx [hunk 2]
```diff
@@ -102,6 +102,34 @@ describe('SocialEmbedContainer', () => {
       unmount();
     });
 
+    it('should render a Twitter block and unmount correctly for STY page', () => {
```
Comment (andrewscfc) [inline src/app/legacy/containers/SocialEmbed/index.test.jsx]: Was this a missing test? Not sure how this relates to this PR

Comment (amoore108) [reply src/app/legacy/containers/SocialEmbed/index.test.jsx]: You're right actually. This folder is a little confusing as there is a CPS subdirectory specifically for CPS usage. The tests in there cover everything that we need, so I'll get rid of this one.

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**